recipes/types-aiobotocore/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "types-aiobotocore" %}
+{% set version = "2.13.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types_aiobotocore-{{ version }}.tar.gz
+  sha256: 88909531df072b6d82b006ce8377388659dab62c8c345c3ded5f17b63589c0f4
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - botocore-stubs
+    - typing-extensions >=4.1.0  # [py<312]
```

We can make this noarch. It should be OK to install typing-extensions on py312.

```suggestion
    - typing-extensions >=4.1.0
```

---

Thanks for suggestion. I just copied verbatim from grayskull 🤣